packages/jsii/test/negatives/neg.behavior-requires-iprefix.ts

```diff
@@ -1,5 +1,3 @@
-///!MATCH_ERROR: Interface contains behavior: name should be "ISomething"
```

Comment:
I'm not seeing this error (or equivalent) in the snapshot. Am I missing it, or did this get dropped somehow?

Reply:
😨 OMG good catch. I guess I was never asserting there is > 0 problems found...

Reply:
Oh and the issue turned out to be essentially that location-less problem markers are NOT rendered with the API style I was going for... Dun dun dun!